docs/urls.md

```diff
@@ -18,11 +18,15 @@ The views would then acessible as:
 - `/api/auth/login` -> `LoginView`
 - `/api/auth/logout` -> `LogoutView`
 - `/api/auth/logoutall` -> `LogoutAllView`
+- `/api/auth/refresh` -> `TokenRefreshView`
+- `/api/auth/refresh/all` -> `TokenRefreshAllView`
```

Comment:
I would rename `logoutall` to `logout/all` to keep things consistent.
As usual adding a note in the changelog that this is backward incompatible and can be restored easily by creating a route manually at the project level.